usaspending_api/common/helpers/spark_helpers.py

```diff
@@ -59,8 +60,20 @@
     DisasterEmergencyFundCode,
     SubmissionAttributes,
     DABSSubmissionWindowSchedule,
+    sam_recipient,
```

Comment:
`from usaspending_api.recipient.models import StateData, DUNS`

and use the `DUNS` model? Feels weird mixing delta models with the postgres django models...

Reply:
Let me know if this change makes sense; not sure if the change I just pushed is what you mean.